R/measures.R

```diff
@@ -64,8 +68,13 @@
 #' }
 riem_measures <- function(
     station = "VOHY",
-    date_start = "2014-01-01",
-    date_end = as.character(Sys.Date())) {
+    data = "all",
+    elev = "no",
+    latlon = "yes",
+    date_start = "2024-01-01",
+    date_end = as.character(Sys.Date()),
+    # skip HFMETAR by default
```

Comment:
why, by the way?

Reply:
First, the [IEM download page](https://mesonet.agron.iastate.edu/request/download.phtml) defaults to routine and specials, but not HFMETAR.  They describe their rationale:

> When in doubt, pick both routine and specials.

It seems to me that IEM is specifying their desired default behavior.  Tangential, I've also found [this](https://mesonet.agron.iastate.edu/onsite/news.phtml?q=HFMETAR) IEM news article.

Second, there is a bandwidth/size penalty (9.1x) and time penalty (6.4x) when selecting HFMETAR.  It seems like an unsafe default.

I compared two single data points:
1. [Link](https://mesonet.agron.iastate.edu/cgi-bin/request/asos.py?station=LAX&data=all&year1=2024&month1=1&day1=1&year2=2024&month2=12&day2=17&tz=Etc%2FUTC&format=onlycomma&latlon=no&elev=no&missing=M&trace=T&direct=no&report_type=3&report_type=4), 2.1 MB in 597ms
2. [Link](https://mesonet.agron.iastate.edu/cgi-bin/request/asos.py?station=LAX&data=all&year1=2024&month1=1&day1=1&year2=2024&month2=12&day2=17&tz=Etc%2FUTC&format=onlycomma&latlon=no&elev=no&missing=M&trace=T&direct=no&report_type=1&report_type=3&report_type=4), 19.1MB in 3820ms

I'm sure our friends at IEM would appreciate the bandwidth savings from users who don't really need HFMETAR.

Third, as a token consumer of this data, my specific application is hampered by the extra time required to download HFMETAR data ... but I'd be happy to set a parameter if I actually needed it.

Reply:
Thank you, it makes sense!